audit/csvformatter_test.go

```diff
@@ -40,8 +40,7 @@ func TestCsvOutputWhenQuiet(t *testing.T) {
 	}
 	entry := Entry{Data: data}
 
-	quiet := true
-	formatter := CsvFormatter{Quiet: &quiet}
+	formatter := CsvFormatter{Quiet: true}
```

Comment:
Might be an unintended consequence, not sure if we want to pass quiet or true all the time (worth a look)

Reply:
> Might be an unintended consequence, not sure if we want to pass quiet or true all the time (worth a look)

The `quiet` default is provided by the Cobra command. We could add a new test of the `loud` mode if desired.
Also, the old `&quiet` reference went away went away.

Sorry if I've totally missed the point.